content/blog/2022/01/multiplatform-kubewarden.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Multiplatform Kubewarden
+authors:
+    - Rafael Fernández López
+date: 2022-02-01
+---
+
+The Kubewarden team is glad to announce that in the spirit of helping
+Policy Authors and Cluster Administrators, the project is now
+officially multiplatform.
+
+The list of supported platforms as of now are:
+
+- Policy Server, as [container image](https://github.com/kubewarden/policy-server/pkgs/container/policy-server):
+  - `linux/amd64` (with the `musl` libc)
+  - `linux/arm64` (with the `musl` libc)
+
+- kwctl, as a [standalone binary](https://github.com/kubewarden/kwctl/releases):
+  - `darwin` (`x86_64`)
+  - `linux` (`aarch64`, with the `musl` libc)
+  - `linux` (`x86_64`, with the `musl` libc)
+  - `windows` (`x86_64`, with `MSVC`)
+
+We have tried to prioritize the same toolchain on platforms where we
+were able to do so.
+
+## Looking for platform convergence
+
+There are two main dependencies in the previous project layout that
+are somewhat special and sometimes problematic to converge to. One is
+`glibc`, and the other is `openssl`.
+
+### The libc
+
+`glibc` only applies to `linux` builds, so in principle is not what
+keeps us from multi platform convergence. However, we thought that
+producing a final static binary for our users in all platforms is
+interesting; specially for a CLI tool like `kwctl`, that our users
+will interact with directly in their environment.
+
+In this spirit, statically compiling a program with `glibc` is tricky
+and can become problematic. The [`musl`](https://musl.libc.org/) libc
+is providing us with this safe, stable and straightforward static
+linking for our binaries targeting linux.
```

Comment:
```suggestion
However, compiling a static binary with `glibc` is tricky and can become problematic, fast. 
The [`musl libc`](https://musl.libc.org/) library is providing us with this safe, stable and straightforward static
linking for our binaries targeting linux.
```

Reply:
as we are talking about multi-platform, maybe add a short sentence about the availability of musl for other platforms?

Reply:
make sense. as the rewording is done, this can be closed.